src/index.html

```diff
@@ -4,9 +4,390 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="catalog">
+      <header class="catalog__header header">
+        <a href="#home" class="header__logo">
+          <img src="images/logo.png" alt="MOYO_logo">
+        </a>
+
+        <nav class="header__nav">
+          <ul class="header__list">
+            <li class="header__item">
+              <a href="#" class="header__link header__link--is-active is-active">Apple</a>
+            </li>
+
+            <li class="header__item">
+              <a href="#" class="header__link">Samsung</a>
+            </li>
+
+            <li class="header__item">
+              <a href="#" class="header__link">Smartphones</a>
+            </li>
+
+            <li class="header__item">
+              <a
+                href="#"
+                class="header__link"
+                data-qa="nav-hover"
+              >Laptops & Computers</a>
+            </li>
+
+            <li class="header__item">
+              <a href="#" class="header__link">Gadgets</a>
+            </li>
+
+            <li class="header__item">
+              <a href="#" class="header__link">Tablets</a>
+            </li>
+
+            <li class="header__item">
+              <a href="#" class="header__link">Photo</a>
+            </li>
+
+            <li class="header__item">
+              <a href="#" class="header__link">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+
+      <main class="catalog__content">
+        <div class="catalog__card card" data-qa="card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+          >
+
+          <h2 class="card__title">APPLE A1419 iMac 27&quot; Retina <br> 5K Monoblock (MNED2UA/A)</h2>
+
+          <p class="card__product-code">Product code: 195434</p>
+
+          <div class="card__rating-container">
+            <ul class="card__stars stars stars--4">
+              <li class="stars__star"></li>
+
+              <li class="stars__star"></li>
+
+              <li class="stars__star"></li>
+
+              <li class="stars__star"></li>
+
+              <li class="stars__star"></li>
+            </ul>
```

Comment:
```suggestion
            <ul class="card__stars stars stars--4">
              <li class="stars__star"></li>
              <li class="stars__star"></li>
              <li class="stars__star"></li>
              <li class="stars__star"></li>
              <li class="stars__star"></li>
            </ul>
```
you should add spaces between multiline blocks, here it's redundant